src/developer/web-api/overview.md

```diff
@@ -86,13 +86,225 @@ using SSL/TLS (HTTPS) to encrypt communication with clients. Consider this
 a hard requirement in order to provide secure interactions with the Web
 API.
 
-### Two-factor authentication { #webapi_2fa } 
+## Two-factor authentication { #webapi_2fa } 
 
 DHIS2 supports two-factor authentication. This can be enabled per user.
 When enabled, users will be asked to enter a 2FA code when logging in. You
 can read more about 2FA [here](https://www.google.com/landing/2step/).
 
-### OAuth2 { #webapi_oauth2 } 
+## Personal Access Token { #webapi_pat_authentication }
+Personal access tokens (PATs) are an alternative to using passwords for
+authentication to DHIS2 when using the API.
+
+PATs can be a more secure alternative to HTTP Basic Authentication,
+and should be your preferred choice when creating a new app/script etc. 
+
+Since HTTP Basic Authentication is considered unsecure because among other things
+that it uses your username and password in clear text, it might be phased out in 
+future versions and made optional, which means it needs to be explicitly enabled 
+in the configuration.
```

Comment:
I'd suggest moving this up to the `basic` authentication section and rephrasing it a little:

```suggestion
HTTP Basic Authentication is considered insecure because, among other things, 
it sends your username and password in clear text. It may be deprecated in 
future DHIS2 versions or made opt-in, meaning that basic authentication would 
need to be explicitly enabled in the configuration.
```

Reply:
Thanks, @amcgee! I agree, this sounds better.